janus_core/helpers/mlip_calculators.py

```diff
@@ -224,6 +224,21 @@ def choose_calculator(
         kwargs.setdefault("sevennet_config", None)
         calculator = SevenNetCalculator(model=model_path, device=device, **kwargs)
 
+    elif arch == "nequip":
+        from nequip import __version__
+        from nequip.ase import NequIPCalculator
+
+        if isinstance(model_path, Path):
+            model_path = str(model_path)
```

Comment:
Should we check it exists as a file? Or does nequip handle it nicely enough?

Reply:
we get 
```bash
ValueError: Please specify `model_path`, as there is no default model for nequip

```

so we are fine